LEAF_Request_Portal/admin/templates/mod_workflow.tpl

```diff
@@ -216,7 +218,9 @@ function removeAllChildren(containerName) {
 }
 
 function insertDOM(elementId, content) {
-    document.getElementById(elementId).innerHTML = content;
+    if(elementId){
```

Comment:
Pretty sure this isn't boolean.

Reply:
added check to make sure this is the proper type.